experimental/addons/argocd/definitions/application.yaml

```diff
@@ -0,0 +1,126 @@
+# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: Argocd-application is a definition of application resource.
```

Comment:
There is no need put both cue and yaml in this dir. The cue file can be translate to x-defn k8s-object
automatically and apply to cluster.

Reply:
I think this is first version, coming up modification may be needed, so i just keep it here first. Should I delete those and commit again?

Reply:
Yes I suggest  keeping the cue files is enough.